### DISCUSSION TITLE
AdGuard Home helper script bug?

### DISCUSSION THREAD

**Discussion by Johnny140** [tteck/Proxmox#3440] [Help Needed] [answered]
Hi. I ran the AdGuard Home helper script today. I entered an ID# not being used by any containers on my system. I get an error stating that VM number is already in use. That is correct, I have a VM with that ID, but no container. I tried 2 more times and get the same error. On the 3rd attempt, I provided a ID that was not used by a current VM or CT and it ran. It created a container with the ID# that I entered.
Thanks.

**Comment by Mellowlynx** [comment, marked as answer]
So you partly answered your question.
An ID needs to be free while making a VM of LXC, there is no difference between a VM and LXC ID, they are the same.
You can't have a VM with the came ID as a LXC or the other way around.
So not script issue.

**Reply by Johnny140** [reply]
Of course, thanks. 
 Doh!!!!!! 
A senior's moment.
